bioio/writers/ome_tiff_writer.py

```diff
@@ -46,6 +47,7 @@ def save(
             Union[List[List[int]], List[Optional[List[List[int]]]]]
         ] = None,
         fs_kwargs: Dict[str, Any] = {},
+        compression: enum.IntEnum = TIFF.COMPRESSION.ADOBE_DEFLATE,
```

Comment:
compression options are missing

Reply:
maybe enough to just say something in the docstring about them and rely on kwargs?